experimental/unitsconversion/src/measureunit.rs

```diff
@@ -23,8 +23,7 @@ pub struct MeasureUnitParser<'data> {
 impl<'data> MeasureUnitParser<'data> {
     // TODO: revisit the public nature of the API. Maybe we should make it private and add a function to create it from a ConverterFactory.
     /// Creates a new MeasureUnitParser from a ZeroTrie payload.
-    #[cfg(feature = "datagen")]
-    pub fn from_payload(payload: &'data ZeroTrieSimpleAscii<ZeroVec<u8>>) -> Self {
+    pub fn from_payload(payload: &'data ZeroTrieSimpleAscii<ZeroVec<'data, u8>>) -> Self {
```

Comment:
Currently we have: `&'data ZeroTrieSimpleAscii<ZeroVec<'data, u8>>`

But we should probably have:

- `&'data ZeroTrieSimpleAscii<ZeroSlice<u8>>`
- `&'data ZeroTrieSimpleAscii<[u8]>`
- `ZeroTrieSimpleAscii<ZeroVec<'data, u8>>`

Reply:
See the following commit:

https://github.com/unicode-org/icu4x/pull/4531/commits/e43a39f96b23ae6a0e42b9af55eee0f26902760e